moduleroot/.travis.yml.erb

```
@@ -88,7 +88,18 @@ branches:
 <%   end -%>
 <% end -%>
 notifications:
-  email: false
+<% notifications_arr = ((@configs['notifications'].to_a || []) - (@configs['remove_notifications'].to_a || [])) -%>
+<% if notifications_arr.any? -%>
```

I see elsewhere in the file we currently use `.any?` without a block as a conditional, but I think if we were applying rubocop rules here it would be:

```suggestion
<% unless notifications_arr.empty? -%>
```

---

Fixed.